torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1092,6 +1115,60 @@ llvm::Error PyTorchModelLoader::loadBatchNorm(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], bn->getResult());
 }
 
+llvm::Error PyTorchModelLoader::loadQuantize(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 4, outputs, 1));
+
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input,
+                             getGlowNodeValue(inputs[QuantizeInputs::input]));
+
+  // scale
+  auto scaleHandle = glow::Handle<float>::createInvalidHandle();
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      scaleHandle, getGlowConstantHandle<float>(inputs[QuantizeInputs::scale]));
+  float outScale = scaleHandle.raw(0);
+
+  // offset
+  auto offsetHandle = glow::Handle<int8_t>::createInvalidHandle();
```

Comment:
just checking on this. is offset int8? (typically it's int32)

Reply:
yeah it is int32...